### PR TITLE
USHIFT-4127: Port upgrade tests to bootc 

### PR DESCRIPTION
**Pull request by ggiguash**
See [this JIRA comment](https://issues.redhat.com/browse/USHIFT-4127?focusedId=25637816&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-25637816) for the list of tests ported and affected.

Note that the following tests were duplicated in presubmits and periodics of the ostree scenarios and they were deleted in the ostree periodics to avoid duplication.
* `el94-crel@el94-src@upgrade-ok.sh`
* `el94-prel@el94-src@upgrade-ok.sh`	
* `el94-yminus2@el94-src@upgrade-ok.sh`

### CODE REVIEW

File: test/scenarios/periodics/el94-crel@el94-src@upgrade-ok.sh [hunk 1]
```diff
@@ -1,32 +0,0 @@
-#!/bin/bash
```
[the remark this answers is not in the document]
Comment (pmtk) [reply test/scenarios/periodics/el94-crel@el94-src@upgrade-ok.sh:1]: Lol, forgot to publish this...

[the remark this answers is not in the document]
Comment (ggiguash) [reply test/scenarios/periodics/el94-crel@el94-src@upgrade-ok.sh:1]: Are you saying we need to keep the "duplication"? Can you explain how you suggest to proceed here?

**Pull request merged**